### ISSUE TITLE
[rush]: rush scan: builtinPackageNames.indexOf is not a function

### ISSUE THREAD

**Issue by alexkreidler** [microsoft/rushstack#2322]
Bug:
```
$ rush scan
Found configuration in /home/alex/c2/loqu/rush.json


Rush Multi-Project Build Tool 5.35.0 - https://rushjs.io
Node.js version is 12.19.0 (LTS)

Found configuration in /home/alex/c2/loqu/rush.json

Starting "rush scan"

Detected dependencies:

ERROR: builtinPackageNames.indexOf is not a function
```

[rush.json](https://hastebin.com/oyikawitep.json)

This happens when I run `rush scan` from any one of my project directories. Please let me know if you need any more info. Thanks!

**Comment by octogonz** [comment]
Thanks for reporting this. I was able to repro it. 

@iclanton This looks like another regression from when we enabled `esModuleInterop`. It's really unfortunate that the compiler cannot catch these problems.

**Comment by octogonz** [comment]
Here's a PR: https://github.com/microsoft/rushstack/pull/2323

**Comment by octogonz** [comment]
Fixed in Rush 5.35.1
